### ISSUE TITLE
Merge Master in React 

### ISSUE THREAD

**Issue by CowMuon** [hicommonwealth/commonwealth#3225]
The top item on our [React Migration Plan](url) is **Merging master into React "still works" with minimum conflicts**

However, somewhere along the way we stopped merging master into react.  :-( 

So now we have a much bigger merge, with a couple hundred merge conflicts. 

That need to be fixed in, so we can ship React.

**Comment by jnaviask** [comment]
Once #3240 is merged, the following commits are outstanding on the React branch:
https://github.com/hicommonwealth/commonwealth/commit/1f7a41c91ec3be56f3a3e939ffc3efeb46ffceb6 (may not be needed, cc @rbennettcw)
https://github.com/hicommonwealth/commonwealth/commit/972b91321be95a39fd62e2b9f26d80582bb5c27c
https://github.com/hicommonwealth/commonwealth/commit/b01b1a549126c2e757065f9e33c92527e5494bae
https://github.com/hicommonwealth/commonwealth/commit/c1baa21fd1c13f23149dbc9f98b78da7d0546c76
https://github.com/hicommonwealth/commonwealth/commit/cb8dbeb1ac60b74bae3e156d4c93baa86a30770b
https://github.com/hicommonwealth/commonwealth/commit/9ce1486678a4e6b3c7be30f46660f0d11e044a21
https://github.com/hicommonwealth/commonwealth/commit/df9b7a05113269b2916acde8c611310bca446e45
https://github.com/hicommonwealth/commonwealth/commit/6e094cca689c2bd436382110d1a258c57359acf6
https://github.com/hicommonwealth/commonwealth/commit/fd14662e029f3d3e92a39cfb71517b476a28dd4b
https://github.com/hicommonwealth/commonwealth/commit/d3a9d0509ca8e130aa8ae16bf23f9ca9f9e7e9ee
https://github.com/hicommonwealth/commonwealth/commit/089d7c0a45ed7afa0dfbef7a6e9d701b9f76588b
https://github.com/hicommonwealth/commonwealth/commit/3ede09915e977d16c765f5598b72f1625401cc1f
https://github.com/hicommonwealth/commonwealth/commit/cbf12586ed768b1049b219d55d5a4220ac70d6c0
https://github.com/hicommonwealth/commonwealth/commit/22a31d1e99ea7087a1751b41150c77b76ccc84f5
https://github.com/hicommonwealth/commonwealth/commit/316062075c5f4336e6d1b149a0a7dbd877d63201
https://github.com/hicommonwealth/commonwealth/commit/1e1d3654e628e50f1f822c0a9296ffcadbb0092d

CET should not go in but need to ensure migrations are up to date against production:
https://github.com/hicommonwealth/commonwealth/commit/68ba0c5c4d3e6a3598862b37e377bcfd29f85bb0
https://github.com/hicommonwealth/commonwealth/commit/12f0d853f12593df1c839e83cbdf55b49044d6d1
https://github.com/hicommonwealth/commonwealth/commit/5f19c8ff356347a54bcb4e8dabbebca830036fe7

**Comment by jnaviask** [comment]
#3240 is merged (I know it says closed, but wanted to preserve history). We should move forward on the above.

**Comment by alexyoung23j** [comment]
- [ ] Delete Polls (https://github.com/hicommonwealth/commonwealth/commit/972b91321be95a39fd62e2b9f26d80582bb5c27c)

**Comment by jnaviask** [comment]
Remaining as of today: everything after Root ID Refactor (https://github.com/hicommonwealth/commonwealth/commit/1e1d3654e628e50f1f822c0a9296ffcadbb0092d)

**Comment by alexyoung23j** [comment]
Probably unnecessary to merge it since it will have been in for 1.5 weeks+ when react goes live, but happy to add if you guys want!

**Comment by CowMuon** [comment]
@alexyoung23j @jnaviask - Can you guys clarify what you mean? Has the email digest and email digest grown been merged into the React branch? (or not?) Also, what about the related minor UI change Jake had to revert?

**Comment by jnaviask** [comment]
Specifically talking about the growl component.
